src/support/path-utils.ts

```diff
@@ -11,31 +11,33 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import {relative} from 'path';
+import {posix, sep as pathSeparator} from 'path';
+
+const dirnameRegex = process.platform === 'win32' ? /[^\\]+$/ : /[^\/]+$/;
 
 /**
  * Similar to `path.dirname()` except includes trailing slash and for a
  * path `/like/this/` will return `/like/this/` instead of `/like` since the
  * trailing slash indicates `this` is a folder name not a file name.
  * (`path.dirname('/like/this/')` returns `/like`.)
  */
-export const dirname = (path: string): string => path.replace(/[^\/]+$/, '');
+export const dirname = (path: string): string => path.replace(dirnameRegex, '');
```

Comment:
`dirname` appears to only operate on OS paths so this regex needs to be platform specific (unlike `getBaseURL` which is basically the same as this function but only operates on URLs so it can assume a `/` separator).